tools/apigee-x-trial-provision/apigee-x-trial-provision.sh

```diff
@@ -74,38 +73,52 @@ export ZONE
 export AX_REGION
 export SUBNET=${SUBNET:-default}
 export ENVOY_MACHINE_TYPE=${ENVOY_MACHINE_TYPE:-e2-micro}
-export ENVOY_PREEMPTIBLE=${ENVOY_PREEMPTIBLE:-true}
-export MANAGED_CERTS=${MANAGED_CERTS:-false}
+export ENVOY_PREEMPTIBLE=${ENVOY_PREEMPTIBLE:-false}
```

Comment:
should we use a different name? There is no `ENVOY` being used.

Reply:
We already do. See the 2nd commit.